### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by seyioyemade**
# Blog app - controllers

In this milestone:

- For each URL I created:

  - A route
  - An action in the correct controller
  - A view file
  
- I created view files that include only placeholders written in plain HTML
- Made sure I added code to handle only the 4 required URLs

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
```
Comment (brytebee) [inline app/controllers/posts_controller.rb:4]: The `show` action is required in the milestone. Kindly add it.

The `show` action is what maps to the show view and the instance variable defined here would be valuable in later projects.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
```
Comment (brytebee) [inline app/controllers/users_controller.rb:4]: The `show` action is required in the milestone. Kindly add it.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @seyioyemade 👋,

##

### Highlights💡
- Great Ruby syntax usage.✔️
- Good commit message.✔️ 
- Good PR Summary.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @seyioyemade,

✔️ Implementation of nested routes

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**